spec/system/page_objects/modals/sidebar_edit_categories.rb

```diff
@@ -3,26 +3,67 @@
 module PageObjects
   module Modals
     class SidebarEditCategories < PageObjects::Modals::Base
-      MODAL_SELECTOR = ".sidebar-categories-form-modal"
```

Comment:
Removing the use of constant as I find that abstracting this away made the selectors harder to read. Even if this does change, the computer has many tools that allows us to easily search and replace.